### PR TITLE
feat(jellyfish-api-core): rpc masternode listanchor

### PR DESCRIPTION
**Pull request by chee-chyuan**
#### What this PR does / why we need it:
Adding rpc listAnchor to masternode
#### Which issue(s) does this PR fixes?:

Fixes #
part of https://github.com/DeFiCh/jellyfish/issues/48

### CODE REVIEW

File: docs/node/CATEGORIES/11-masternode.md [hunk 1]
```diff
@@ -199,3 +199,26 @@ interface masternode {
   getActiveMasternodeCount (blockCount: number = 20160): Promise<number>
 }
 ```
+
+## listAnchors
+Return array of anchors if any
```
Comment (surangap) [inline docs/node/CATEGORIES/11-masternode.md:204]:
```suggestion
Returns an array of anchors if any
```

File: docs/node/CATEGORIES/11-masternode.md [hunk 2]
```diff
@@ -199,3 +199,26 @@ interface masternode {
...
+
+```ts title="client.masternode.getActiveMasternodeCount"
```
Comment (surangap) [inline docs/node/CATEGORIES/11-masternode.md:206]: ??

Comment (chee-chyuan) [reply docs/node/CATEGORIES/11-masternode.md:206]: oh my what was i doing here

File: packages/jellyfish-api-core/src/category/masternode.ts [hunk 3]
```diff
@@ -193,6 +193,14 @@ export class Masternode {
   async getActiveMasternodeCount (blockCount: number = 20160): Promise<number> {
     return await this.client.call('getactivemasternodecount', [blockCount], 'number')
   }
+
+  /**
+   * Returns array of anchors if any
+   * @return {} Array of anchors
```
Comment (surangap) [inline packages/jellyfish-api-core/src/category/masternode.ts:199]:
```suggestion
   * Returns an array of anchors if any
   * @return Promise<MasternodeResult<MasternodeAnchor>>
```

**Review by surangap — COMMENTED** [review]
looks good

**Review by surangap — DISMISSED** [review]

**Review by canonbrother — APPROVED** [review]

**Review by fuxingloh — DISMISSED** [review]

**Comment by fuxingloh** [comment]
Conflicts

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**